compiler/rustc_hir_typeck/src/demand.rs

```diff
@@ -954,6 +955,85 @@ impl<'a, 'tcx> FnCtxt<'a, 'tcx> {
         );
     }
 
+    pub(crate) fn suggest_missing_unwrap_expect(
```

Comment:
can you move this to `compiler/rustc_hir_typeck/src/fn_ctxt/suggestions.rs`?

Reply:
moved it.

Reply:
seems there are several other functions we may need to move :)